polling_stations/apps/data_collection/management/commands/import_aberdeen.py

```diff
@@ -10,11 +10,12 @@
 class Command(BaseScotlandSpatialHubImporter):
     council_id = "S12000033"
     council_name = "Aberdeen City"
-    elections = ["local.aberdeen-city.2017-05-04", "parl.2017-06-08"]
+    elections = ["europarl.2019-05-23"]
+    private = True
```

Comment:
This shouldn't be needed. `BaseScotlandSpatialHubImporter.data_path()` should respect the `PRIVATE_DATA_PATH` setting